angular_ast/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 0.4.0-alpha+1
+- New code location! angular_ast is now part of the angular mono-repo on
+  https://github.com/dart-lang/angular.
+### Bug fix
```

Comment:
Add a line above this section head. (Not sure whether it'll render as-is, but the blank line seems like a good practice anyway.)

Reply:
Just confirmed that it *does* render OK on both github and https://dart-lang.github.io/markdown/, but still...

Reply:
Done